fullscreen/api/element-request-fullscreen.html

```diff
@@ -18,6 +21,7 @@
         t.done();
     }));
 
-    trusted_request(t, div);
+    test_driver.bless("fullscreen", () => div.requestFullscreen())
+        .catch(t.step_func(assert_unreached));
```

Comment:
`t.unreached_func(…)`

Reply:
Unfortunately, to include the reason it would have to be `.catch(reason => t.unreached_func(reason)())` which I thought was a lot worse.